### PR TITLE
Release notes 2.8 and update of readme

### PR DESCRIPTION
**Pull request by MKodde**
This PR consists of two documentation changes:
1. The new 2.8.0 changelog entry
1. Update of the readme advertising the new Docker based dev env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,27 +60,18 @@ ANSIBLE_TAGS=eb,profile vagrant provision
...
- 
 The application is now up and running and can be accessed at
-[https://spdashboard.dev.support.surfconext.nl/](https://spdashboard.dev.support.surfconext.nl/). Note that in development the `app_dev.php`
+[https://spdashboard.vm.openconext.org/](https://spdashboard.vm.openconext.org). Note that in development the `app_dev.php`
 front controller is used automatically, so you don't have to include `/app_dev.php/` in the URLs.
-* To view mails caught by Mailcatcher, visit [spdashboard.dev.support.surfconext.nl:1080](https://spdashboard.dev.support.surfconext.nl:1080/)
-
-If you run into the `shibsp::ConfigurationException`, please reload your box, the issue should be resolved after a 
-reboot. The Shiboleth deamon might not come out 100% correctly out of the initial provisioning run.
+* To view mails caught by Mailcatcher, visit [spdashboard.vm.openconext.org:1080](https://spdashboard.vm.openconext.org:1080/)
```
Comment (MKodde) [inline README.md:74]: I have not yet tested if Mailcatcher actually works. @quartje have you?

File: README.md [hunk 2]
```diff
@@ -10,24 +10,39 @@
...
+- [Docker Compose](https://docs.docker.com/compose/install/)
 
+As of release 2.8 the Vagrant + Ansible dev environment has been discontinued in favour of a docker-compose installable
+dev machine. Instructions below should still help you in building a Vagrant based dev env, but this will be removed from
+the project in the next release. For now use `docker-compose up -d` to create and build the development environment.
+
+An entry in your hostsfile is still required for things to work. An example entry would look like:
+
+```
+127.0.0.1 welcome.vm.openconext.org static.vm.openconext.org mujina-sp.vm.openconext.org mujina-idp.vm.openconext.org engine-api.vm.openconext.org oidc.vm.openconext.org manage.vm.openconext.org spdashboard.vm.openconext.org
```
Comment (MKodde) [inline README.md:33]: @quartje which apps actually run on the base openconext docker image? I stripped some that I knew where not on there, but can you further validate this hostsfile entry?

**Review by MKodde — COMMENTED** [review]
@quartje and @Badlapje please do not be gentle in reviewing this. I might have missed some interesting release note entries or maybe you guys know more stuff for the readme? Shoot to kill!

File: README.md (hunk 2 above)
Comment (quartje) [reply README.md:33]: I'll open another PR to document the Docker changes.

**Pull request merged**